lib/foreman_snapshot_management/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ForemanSnapshotManagement
-  VERSION = '3.0.0'
+  VERSION = '3.1.0'
```

Comment:
```suggestion
  VERSION = '3.0.1'
```
I don't see any new features in the commits since the bump to 3.0.0, so new version should be 3.0.1 IMHO. Can you please verify?